simulation/multi_agent_simulation.py

```diff
@@ -36,8 +36,8 @@ def main():
     ffmpeg_process = None
 
     # Environment x.
-    min_x = 1e-10
-    max_x = 5e-6
+    min_x = 1e-8
+    max_x = 8e-5
```

Comment:
we really need to move that to a config file or sth - not in this PR though

Reply:
:100:

Reply:
#11